src/Shouldly.Tests/ShouldNotContain/StringContainsStringCaseIsInsensitiveScenario.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.ShouldNotContain
+{
+    public class StringContainsStringCaseIsInsensitiveScenario : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Shouldly is legendary".ShouldNotContain("legendary", Case.Insensitive);
```

Comment:
Then you can also use

```
public override bool CanProcess(IShouldlyAssertionContext context)
{
    return context is ExpectedActualCaseInsensitiveShouldlyContext;
}
```

Rather than reflection